src/app/components/calendar.js

```diff
@@ -0,0 +1,71 @@
+import axios from 'axios';
+
+export default function (container) {
+    const boxes = [...container.querySelectorAll('.js-box')];
+    const toggleBoxes = [...container.querySelectorAll('.js-toggler')];
+    const toggleTexts = [...container.querySelectorAll('.js-toggler-text')];
+    const boxIcons = [...container.querySelectorAll('.js-icon-box')];
+    const reminderBtn = container.querySelector('.js-reminder-submit');
+    const reminder = container.querySelector('.js-reminder');
+    const mailTimeBtn = container.querySelector('.js-mail-time-submit');
+    const mailTime = container.querySelector('.js-mail-time');
+    const resetBtn = container.querySelector('.js-reset-submit');
+    const active = 'active';
+    const inactive = 'inactive';
+    const hide = 'v-hide';
+
+    toggleBoxes.forEach((toggler) => {
+        const box = boxes.find(box => box.dataset.dayBox === toggler.dataset.dayBtn);
+        const innerText = toggleTexts.find(text => text.dataset.dayText === toggler.dataset.dayBtn);
+        toggler.addEventListener('click', () => {
+            boxToggle(box);
+            postRequest(String(box.dataset.dayBox), String(box.classList.contains(active)));
```

Comment:
in 43rd row, you pass the second argument as bool, why you here pass as a string.
everything in dataset is a string